haiku-apps/voxophone/voxophone-1.0.recipe

```diff
@@ -0,0 +1,44 @@
+SUMMARY="An online radio player"
+DESCRIPTION="Voxophone is a online radio player application for Haiku."
+HOMEPAGE="https://github.com/aarroyoc/Voxophone"
+COPYRIGHT="2015 Adrian Arroyo Calle"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="67514047957bb8a186673ccc6fa71d2d0a1e1728"
+SOURCE_URI="https://github.com/aarroyoc/Voxophone/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="bc98a70248057aa482f038e3689876dc0632aca4aa65d2417d9f692c86b486b5"
+SOURCE_FILENAME="voxophone-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="Voxophone-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	voxophone = $portVersion
+	app:Voxophone = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:cmake
+	cmd:g++
+	cmd:make
+	"
+
+BUILD()
+{
+	cmake .
+	make
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir
+	cp Voxophone $appsDir
```

Comment:
> Finally back to recipes 😉

Cool!

Suggestion:
```diff
-	mkdir -p $appsDir
+	install -d $appsDir
-	cp Voxophone $appsDir
+	install Voxophone $appsDir
```

BTW, I'm trying to build it on x86_64, but it needs libnetapi, which is provided by samba.
So I think you should add `lib:libnetapi` do `REQUIRES` and `devel:libnetapi` do `BUILD_REQUIRES`. (I'll try that.)

Reply:
Good idea, @fbrosson 😃 
I'll also add those requirements.

Reply:
@fbrosson are you sure? Libnetapi isn't the network API on R5?

Reply:
Sorry @korli, these messages are outdated. I should have stroken the text.
The explanation is [here](https://github.com/haikuports/haikuports/pull/2082#discussion_r160065744).